src/data/link-cards-data.ts

```diff
@@ -1,46 +1,51 @@
 //these values are just placeholders for now will be updated by content authors
 const linkCardData = {
   android: {
-    github: '#',
-    roadmap: '#',
+    github: 'https://github.com/aws-amplify/amplify-android',
+    roadmap: undefined,
```

Comment:
Rather than setting as undefined, prefer to leave off the prop
```suggestion
```

Reply:
okay making it optional, but I like keyword `undefined` :) , thanks